### PR TITLE
Add multiple icon to MenuItem

### LINKED ISSUE #959: Add multiple icon to MenuItem

### PR DESCRIPTION
**Pull request by DenisVorop**
## PR includes

- [x] Refactor
- [x] Feature

## Related issues

Resolve #959 

Related issues:
- https://github.com/taskany-inc/issues/pull/2419

## QA Instructions, Screenshots, Recordings
  
Снимок экрана 2024-03-13 в 19 33 30
Снимок экрана 2024-03-13 в 19 33 26

### CODE REVIEW

File: src/harmony/MenuItem/MenuItem.tsx [hunk 1]
```diff
@@ -22,18 +26,22 @@ export const MenuItem = ({
...
             {nullable(
                 selectable && !iconLeft,
-                () =>
-                    nullable(
-                        selected,
-                        () => <IconTickOutline size={size} />,
-                        <BaseIcon value={() => null} size={size} />,
-                    ),
+                () => (
+                    <BaseIcon value={() => nullable(selected, () => icon)} size={multiple ? size : dotSize} />
```
Comment (awinogradov) [inline src/harmony/MenuItem/MenuItem.tsx:43]: Why u need BaseIcon here?

Comment (DenisVorop) [reply src/harmony/MenuItem/MenuItem.tsx:43]: To ensure that elements without icons are aligned relative to elements with icons

Comment (awinogradov) [reply src/harmony/MenuItem/MenuItem.tsx:43]: BaseIcon is the only one way to align elements in MenuItem?

Comment (DenisVorop) [reply src/harmony/MenuItem/MenuItem.tsx:43]: each MenuItem element does not know which item is selected and with which icon (multiple or not)

baseIcon ensures the correct indentation

**Review by awinogradov — APPROVED** [review]

**Pull request merged**